app/User.php

```diff
@@ -52,4 +52,11 @@ public function getPermissionsAttribute()
     {
         return $this->roles()->pluck('permissions')->collapse();
     }
+
+    public function verified()
```

Comment:
ok ho capito, l'ho modificato in verify, ed il check che verifica se è verificato (che si chiamava a sua volta verify, l'ho chiamato ifverify)